README.adoc

```diff
@@ -4000,6 +4000,23 @@ When you've got keys that are not symbols stick to the hash rockets syntax.
 { :a => 1, 'b' => 2 }
 ----
 
+=== Array to hash [[array-to-hash]]
```

Comment:
For the same reason we discussed in Rubocop's PR, I believe "array" shouldn't be here :)
Something like "Don't use `Hash[]` constructor", or something.

Reply:
changed @zverok